src/pages/ws/ui/variant-drawer/variant-drawer-header/drawer-note/drawer-note.tsx

```diff
@@ -42,6 +42,7 @@ export const DrawerNote = observer(() => {
         isOpen={!isLoading && !!popoverAnchorEl}
         anchorEl={popoverAnchorEl}
         onClose={closePopover}
+        key={noteText}
```

Comment:
i think, its better to use variant._no as key

Reply:
Done